src/controllers/tileController.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import { AppLocals } from "../types/app";
+
+export class TileController {
+    static getTile = async (req: Request, res: Response) => {
+        const { dataset, level, z, x, y } = req.params;
+        const { tileDatasets } = req.app.locals as AppLocals;
+
+        let tileData = null;
+        if (tileDatasets[dataset] && tileDatasets[dataset][level]) {
+            const db = tileDatasets[dataset][level];
+            tileData = await db.getTileData(
+                parseInt(z),
+                parseInt(x),
+                parseInt(y)
```

Comment:
Error handling for bad requests where these values aren't parseable to be done in https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6085/Error-handling